google/cloud/spanner_dbapi/batch_executor.py

```diff
@@ -61,7 +106,7 @@ def execute_statement(self, parsed_statement: ParsedStatement):
             raise ProgrammingError("Only DML statements are allowed in batch DML mode.")
         self._statements.append(parsed_statement.statement)
 
-    def run_batch_dml(self):
+    def run_batch(self):
```

Comment:
Ack. Added the following documentation on the method `This method is internal and not for public use as it can change anytime.`

Reply:
Instead of renaming the method, we could mark it as deprecated and create a new one with the `_` prefix which indicates that it is internal. Please avoid a major version bump if at all possible as it is disruptive for downstream users.

Reply:
This is not the reason because of which we are proposing a major version bump. Please check https://docs.google.com/document/d/1mSVC_AhPpdSA0xUoj4LUt2o2tVM_LDv6IdssiwWnbmw for details